packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -2255,6 +2260,23 @@ function getRetryCache(finishedWork) {
   }
 }
 
+function attachOffscreenActions(offscreenFiber: Fiber, root: FiberRoot) {
+  offscreenFiber.stateNode.detach = () => {
```

Comment:
This is a new method every single time. We should try to avoid this.
Maybe store current fiber on stateNode and reference that instead of offscreenFiber directly.